mozbench/mozbench.py

```diff
@@ -36,6 +36,105 @@
 headers = None
 results = None
 
+class ResultRecorder(object):
+
+    def __init__(self):
+        self._platform = 'unknown'
+        self._osVersion = 'unknown'
+        self._processor = 'unknown'
+        self._currentBrowser = None
+        self._currentBenchmark = None
+        self._browsers = {}
+
+    def setPlatformName(self, name):
+        self._platform = name
```

Comment:
The python style is to not write simple getter/setter methods like this. Please just have a self.platform member and remove these functions.